Work around a recently exposed automake bug producing ar warnings

  ar: `u' modifier ignored since `D' is the default

... on Ubuntu 16.04 and other "modern" systems.

See also:
  https://github.com/kimwalisch/primesieve/issues/16
  https://bugzilla.redhat.com/show_bug.cgi?id=1155273#c12

diff --git b/configure.ac a/configure.ac
index ddb442e..17835f7 100644
--- b/configure.ac
+++ a/configure.ac
@@ -51,6 +51,11 @@ dnl check for programs
 AC_PROG_CXX
 AC_PROG_CXXCPP
 AC_PROG_INSTALL
+
+dnl A temporary workaround to silence ar warning:
+dnl `u' modifier ignored since `D' is the default
+AC_SUBST(ARFLAGS, [cr])
+AC_SUBST(AR_FLAGS, [cr])
 AM_PROG_AR
 
 
